src/assets/css/librarybrowser.css

```diff
@@ -952,6 +952,10 @@ div.itemDetailGalleryLink.defaultCardBackground {
     flex-wrap: wrap;
 }
 
+.verticalSection {
+    animation: fadein 0.2s;
```

Comment:
I guess it already existed somewhere and you're just adding animations, however let me ask: where it's used?

Why not a *slow* 0.7 variation?

Reply:
> Why not a slow 0.7 variation?

It's not supposed to be 0.2s, because that still looks like it pops in :smile: 

The goal was to make the sections fade-in as a way to add the egregious way that sections get added to pages.